src/compas_fab/ghpython/artists.py

```diff
@@ -24,7 +24,7 @@ def transform(self, native_mesh, transformation):
         T = xform_from_transformation(transformation)
         native_mesh.Transform(T)
 
-    def draw_mesh(self, compas_mesh, color=None):
+    def draw_geometry(self, compas_mesh, color=None):
```

Comment:
same comment, maybe instead of `compas_mesh` this should be called something else if it accepts more than just meshes?

Reply:
changed to `compas.datastructures.Mesh` or `compas.datastructures.Shape` in description